app/src/main/java/com/github/se/cyrcle/model/map/MapViewModel.kt

```diff
@@ -205,15 +210,13 @@ class MapViewModel : ViewModel() {
   }
 
   /**
-   * Get the bottom left and top right corners of the screen in latitude and longitude coordinates.
-   * The corners are calculated based on the center of the screen and the viewport dimensions. If
-   * useBuffer is true, the corners are calculated with a buffer of 2x the viewport dimensions. This
-   * is useful for loading parkings that are not yet visible on the screen.
+   * Updates the bottom left and top right corners of the screen in latitude and longitude
```

Comment:
Like the new documentation! Much more concise :)

Reply:
I agree with Louis on this one, documentation is a lot clearer !